src/mono/wasm/runtime/http.ts

```diff
@@ -21,7 +21,9 @@ export function http_wasm_abort_request(abort_controller: AbortController): void
 export function http_wasm_abort_response(res: ResponseExtension): void {
     res.__abort_controller.abort();
     if (res.__reader) {
-        res.__reader.cancel();
+        res.__reader.cancel().catch(() => {
+            //we ignore the error
```

Comment:
Why? Even logging e.g. "Reading the steam was canceled" is not desirable?

Reply:
It's the direct consequence of the `__abort_controller.abort()` above. 
The user is asking for it. It's expected to happen.

We could possibly test if the error is something else than we expected and log that.